Generator/Generator/MethodGen.swift

```diff
@@ -30,6 +30,10 @@ enum GeneratedMethodKind {
     case utilityFunction
 }
 
+/// The list of static method names to make public.
+/// Most of the names are kept fileprivate.
+let publicMethodNames = ["method_get_class", "method_emit_signal"]
```

Comment:
I had to make this public in order to implement an alternative version of `emitSignal`, which the `GenericSignal` implementation needs in order to implement an `emit()` method with the correct arguments. See also `RawCall.swift`.

I suspect that there may be a better way. Possibly I can use a Variant to make the call?